README.md

```diff
@@ -213,6 +214,15 @@ async def aresponses(loop):
         yield server
 ```
 
+If you're trying to use the `aiohttp_client` test fixture then you'll need to mock out the aiohttp `loop` fixture
+instead:
+```python
+@pytest.fixture
+def loop(event_loop):
+    """replace aiohttp loop fixture with pytest-asyncio fixture"""
+    return event_loop
+```
+
```

Comment:
More importantly, this needs to go in a proper documentation page. I'd not consider a Git/GitHub README to be part of the things you *have* to read to use a library.

Reply:
For now, the readme is our only documentation.  Changing that would be outside the scope of this PR.

Reply:
Ah, lol I did not even realize that until now sorry.